scripts/release/index.ts

```diff
@@ -0,0 +1,150 @@
+  const packageSubtype = getPackageSubtype(pckg);
+  const list: PACKAGE_SUBTYPE[] = [];
+  if (packageSubtype === PACKAGE_SUBTYPE.INDEPENDENT) {
+    return list;
+  }
+
+  const enumKeys = Object.keys(PACKAGE_SUBTYPE).filter(key => isNaN(Number(key)));
+  for (let i = packageSubtype + 1; i < enumKeys.length; i++) {
+    list.push(i);
+  }
+
+  return list;
+};
+
+const program = () => {
+  const args = process.argv.slice(2);
+  let pckg = args[0];
+  // args[1] === '-'
+  const gradation = args[2];
+
+  if (!pckg || !gradation) {
+    console.log('Provide package and gradation. Pattern is: {package} - {gradation}');
+    return;
+  }
+  if (!isProperGradation(gradation)) {
+    console.log(`${gradation} is bad gradation. Use 'path', 'minor' or 'major'`);
+    return;
+  }
+
+  const packageType = getPackageType(pckg);
+  if (!packageType) {
+    console.log('Bad package\'s type');
+    return;
+  }
+
+  let operationList = {
+    pathsToRun: [],
+    freshVersions: {},
+    oldFiles: {}
+  };
+  let customPackagesInIntegrationWrapper = [];
+
+  if (packageType === PACKAGE_TYPES.Package || packageType === PACKAGE_TYPES.IntegrationWrapper) {
+    if (packageType === PACKAGE_TYPES.IntegrationWrapper) {
+      customPackagesInIntegrationWrapper = getCustomPackagesInIntegrationWrapper(pckg);
+      pckg = `${pckg}/api-client`;
+    }
+    const dependencyList: PACKAGE_SUBTYPE[] = buildPackageDependencyList(pckg);
+    try {
+      operationList = updatePackageVersion(pckg, RELEASE_GRADATIONS[gradation]);
+      for (const subtype of dependencyList) {
+        operationList = updatePackageVersion(pckgSubtypeToPath(pckg, subtype), RELEASE_GRADATIONS[gradation], operationList);
+      }
+      for (const customPackage of customPackagesInIntegrationWrapper) {
+        operationList = updatePackageVersion(pckgCustomSubpackageToPath(pckg, customPackage), RELEASE_GRADATIONS[gradation], operationList);
+      }
+    } catch (err) {
+      console.log(err);
+    }
+
+    const statusLog = [];
+
+    try {
+      while (operationList.pathsToRun.length) {
+        publishPackage(operationList.pathsToRun[0]);
+        statusLog.push('Succesfully published ' + sliceAbsolutePathPart(operationList.pathsToRun[0]) + ` new ${gradation} version`);
+        operationList.pathsToRun.shift();
+      }
+    } catch (err) {
+      for (const pathToRun of operationList.pathsToRun) {
+        statusLog.push('Failed publishing ' + sliceAbsolutePathPart(pathToRun));
+        const pathWithFile = `${pathToRun}/package.json`;
+        fs.writeFileSync(pathWithFile, JSON.stringify(operationList.oldFiles[pathWithFile], null, 2));
+        statusLog.push('Rolled back to old version');
+      }
+    }
+
+    console.log('\n');
+    for (const log of statusLog) {
+      console.log(log);
+    }
+
+  } else if (packageType === PACKAGE_TYPES.Wrapper && pckg === 'core') {
+    console.log('Pick specific core\'s package');
```

Comment:
@filrak shouldn't we use `consola` for such a thing?

Reply:
we can keep console.log - this script is just for us

Reply:
We would prefer `consola` normally. However, as this script will be only for Core Team - it does not matter